egs/csj/s5/local/csj_make_trans/csj_automake.sh

```diff
@@ -5,113 +5,165 @@
 # Apache 2.0
 # Acknowledgement  This work was supported by JSPS KAKENHI Grant Number 26280055.
```

Comment:
I think it would be better to rename to csj_autorun.sh.
automake is the name of a GNU tool for Makefile generation.